pkgs/development/python-modules/klaus/default.nix

```diff
@@ -52,6 +52,8 @@ buildPythonPackage rec {
   # Needs to set up some git repos
   doCheck = false;
 
+  patches = [ ./distutils.patch ];
```

Comment:
any reasons not to use fetchpatch instead ?

Reply:
Wasn't aware it existed. I've updated to use fetchpatch